client/src/components/Admin/IssueForm/index.js

```diff
@@ -101,13 +101,19 @@ class IssueForm extends React.Component {
           hidden={!showActiveIssueWarning}
         >Det er allerede en aktiv sak!</p>
         <label className="IssueForm-textarea">
-          <div className="IssueForm-label">Beskrivelse</div>
+          <h4 className="IssueForm-label">Beskrivelse av saken</h4>
```

Comment:
No, I don't mean the size. 

h1, h2 etc have a semantic meaning and tools that check for accessibility will cry about using them out of order. 
If the size is too small/big just set font-size. It's better than relying on the default browser size imo.

http://khan.github.io/tota11y/ is nice for finding some simple things like low contrast and misuse of h1, h2.